Edit: Getting coins (#75)
FalixNodes-Software/help-center: _posts/falix/general/2021-11-02-getting-coins.md

permalink: /falix/general/getting-coins/
---

> Curently there is only one way of getting coins, you can find it [here](https://client.falixnodes.net/coins).

Reviewer: ```suggestion
Curently there is only one way of getting FalixCoins, you can find it [here](https://client.falixnodes.net/coins).
```

4. Click on register.
![image](../../../assets/images/posts/falix/earning-coins/using-affiliate-code.png)

> Note: Alting is **strictly prohibited**.
-->
# Joining Discord Servers

> Make sure your Discord account is connected to your FalixNodes account, you can find out how by following the [Account Management guide](https://help.falixnodes.net/falix/general/account-management/).

Reviewer: ```suggestion
Make sure your Discord account is connected to your FalixNodes account. You can do so by following the [Account Management guide](https://help.falixnodes.net/falix/general/account-management/).
```

1. Go to the [Join For Rewards](https://client.falixnodes.net/join_for_rewards) page.

> 2. You should see a list of servers on your right, click on "join" to join them.

Reviewer: ```suggestion
2. You should see a list of servers on the right. Click on the "join" button next to a server to join it.
```

> 3. To receive coins for joining servers, click on "Get FalixCoins For Joined Servers", the button is located above the server list.

Reviewer: ```suggestion
3. To receive coins for joining servers, click on "Get FalixCoins For Joined Servers", the button is located above the server list. If this does not work, make sure to try relogging (logging out and logging back in).
```
